tests/test_cli/test_cli.py

```diff
@@ -213,5 +219,286 @@ def test_unpack_characterizations_no_overwrite(
         assert isinstance(result.exception, FileExistsError)
 
 
+@pytest.mark.filterwarnings("ignore:Skipping")
+@pytest.mark.filterwarnings("ignore:Geometry is in a geographic:UserWarning")
+def test_make_maps_solar(
+    test_map_supply_curve_solar, test_cli_runner, test_data_dir
+):
+    """
+    Happy path test for make_maps() CLI. Tests that it produces the expected
+    images for a solar supply curve.
+    """
+
+    with tempfile.TemporaryDirectory() as tempdir:
+        output_path = pathlib.Path(tempdir)
+        result = test_cli_runner.invoke(
+            make_maps, [
+                '-i', test_map_supply_curve_solar.as_posix(),
+                '-t', "solar",
+                '-o', output_path.as_posix(),
+                '--dpi', 75
+            ]
+        )
+        assert result.exit_code == 0
+
+        out_png_names = [
+            "capacity_solar.png",
+            "lcot_solar.png",
+            "mean_lcoe_solar.png",
+            "total_lcoe_solar.png"
+        ]
+        for out_png_name in out_png_names:
+            expected_png = test_data_dir.joinpath("plots", out_png_name)
+            out_png = output_path.joinpath(out_png_name)
+            images_match_exactly = compare_images_exact(expected_png, out_png)
+            if not images_match_exactly:
+                assert compare_images_approx(expected_png, out_png), \
```

Comment:
Is there any reason not to just test the `compare_images_approx`? If the test passes you have no way of knowing which condition was met anyways (and same comment for below)